CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ cmake_policy(SET CMP0091 NEW) # enables MSVC_RUNTIME_LIBRARY target property
 # Versions are now in the form x.y.z
 # Changed 1.0 to 2.0 because API is extended with virtual ports:
 set(SOVERSION "2")
-set(VERSION "2.0.4")
+set(VERSION "2.0.7")
```

Comment:
Is 2.0.7 the right number? I searched for 2.0.5, 2.0.6 and found nothing. It's not critical, and quite possibly there was an update after 2.0.4 without a version number change, but I'm just curious.

Reply:
@rbdannenberg I set this number for the next release in advance

Reply:
Thanks - somehow, I didn't get your note until I looked just now, but independently saw there is a release 2.0.6, which I can't really explain, but your are right: it should be 2.0.7. Pulling the request now. Thanks.